sda-commons-server-s3/src/main/java/org/sdase/commons/server/s3/S3Bundle.java

```diff
@@ -113,6 +139,7 @@ public static InitialBuilder builder() {
   public static class Builder<T extends Configuration> implements InitialBuilder, FinalBuilder<T> {
 
     private final S3ConfigurationProvider<T> configProvider;
+    private boolean healthCheckDisabled = false;
```

Comment:
The commit message (and therefore the announced feature) mentions that an external health check is added.
For me it looks like a regular internal health check is introduced that can be turned into an external health check.
What is the impact on existing services when a new health check is introduced by default?

Is this a breaking change, @SDA-SE/sda-commons-maintainer ?
If not, I would say the change is good as is, but the commit message should be changed.
Otherwise, I would change the default to external.